.github/workflows/ci.yml

```diff
@@ -1,5 +1,8 @@
 name: CI
 
+env:
+  DEVNET_TAG: "v0.2.2"
+
```

Comment:
We should use commit hash because it is immutable, unlike a tag which is mutable therefore could theoretically pose some risk.

Reply:
Updated here as well